### ISSUE TITLE
[idea] Setting to configure word regex.

### ISSUE THREAD

**Issue by trusktr** [t9md/atom-quick-highlight#9]
For example, I am using this, and would like to match the regex that `vim-mode` uses for `*` and `#`. Possibly would be nice to add option to match vim-mode's regex, or to allow a custom regex.

**Comment by t9md** [comment]
Yeah, I have this idea from start, this package is port of my [vim-quickhl](https://github.com/t9md/vim-quickhl) to Atom.
Original vim version have regex option and static pattern(always highlighted keywords without manual operation).

It's good to have, but I have many thing to do in other project, so if someone sent PR I'll welcome to look into.
Or else, I can implement if I have time and want some refreshment.
Thanks for suggestion.
